rasa/core/channels/botframework.py

```diff
@@ -174,6 +176,17 @@ async def send_custom_json(
         ).setdefault("alert", "true")
         json_message.setdefault("text", "")
         await self.send(json_message)
+    
+    @staticmethod
+    def _add_button_metadata(button: Dict[Text, Any]) -> Dict[Text, Any]:    
+        button.update(
+            {
+                "type": "messageBack",
+                "value": button.get("payload"),
+                "text": button.get('title'),
```

Comment:
Shouldn't the key name for this value actually be `title`?
According to the example in the docs you shared, the value given to "text" can be a hardcoded string?